### PR TITLE
Script fixes

### PR DESCRIPTION
**Pull request by aviadl**

### CODE REVIEW

File: Scripts/script-AddEvidence.yml [hunk 1]
```diff
@@ -17,11 +17,9 @@ script: |-
       if (isValidRes(entries)) {
           for (var j=0;j<entries.length;j++){
               var ent=entries[j];
-              if (((ent.ContentsFormat == 'table' || ent.ContentsFormat == 'markdown') && ent.Contents) || ent.Type == entryTypes['file']) {
-                  var res = executeCommand('markAsEvidence', {'id': entryID, 'description': desc});
-                  if (!isValidRes(res)) {
-                      return res;
-                  }
+              var res = executeCommand('markAsEvidence', {'id': entryID, 'description': desc});
```
Comment (slavikm) [inline Scripts/script-AddEvidence.yml]: I think the idea here was to ignore simple text responses like `yes` or `no` which might be part of the array of last entries

File: Scripts/script-CommonServer.yml [hunk 2]
```diff
@@ -1001,6 +1001,30 @@ script: |-
...
+
+      var parts = subnet.split('/');
+      if (parts.length == 1) {
+          return ip == parts[0];
+      }
+      if (IPnumber(parts[0]) && parts[1].match(/^(\d+)/)) {
+          return (IPnumber(ip) & IPmask(parts[1])) == IPnumber(parts[0]);
+      }
+      return false;
+  };
```
Comment (slavikm) [inline Scripts/script-CommonServer.yml]: no need for `;`

File: Scripts/script-IsIPInSubnet.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+script: 'return isIPInSubnet(args.ip, args.subnet) ? ''yes'' : ''no'''
+type: javascript
+tags:
+- Condition
+enabled: true
+args:
+- name: ip
+  required: true
+  default: true
+  description: Ip to check
```
Comment (slavikm) [inline Scripts/script-IsIPInSubnet.yml]: Ip -> IP

File: Scripts/script-AddEvidence.yml (hunk 1 above)
Comment (aviadl) [inline Scripts/script-AddEvidence.yml]: today it is not relevant anymore

**Pull request merged**